ansible/roles/index-signature-verification/files/tasks/verify-index-signatures.yml

```diff
@@ -36,6 +39,9 @@ spec:
     - name: keys
       configMap:
         name: "$(params.keys_config_map)"
+    - name: policies
```

Comment:
How does it know where to mount the volume? Other volumes are mounted to a specific location, but this one isn't.

Reply:
Signature verification now works. I update this Pull Request with the working changes.